packages/cli/src/commands/org/config/push.ts

```diff
@@ -1,25 +1,39 @@
 import {cli} from 'cli-ux';
-import {flags} from '@oclif/command';
-import {ReadStream} from 'fs';
+import {flags, Command} from '@oclif/command';
 import {
   IsAuthenticated,
   Preconditions,
 } from '../../../lib/decorators/preconditions';
 import {Snapshot} from '../../../lib/snapshot/snapshot';
 import {red, green, bold} from 'chalk';
-import SnapshotBase from './orgConfigBase';
 import {SnapshotReporter} from '../../../lib/snapshot/snapshotReporter';
+import {
+  displayInvalidSnapshotError,
+  displaySnapshotSynchronizationWarning,
+  dryRun,
+} from '../../../lib/snapshot/snapshotCommon';
+import {Config} from '../../../lib/config/config';
+import {DryRunOptions} from '@coveord/platform-client';
+import {cwd} from 'process';
 
-export interface CustomFile extends ReadStream {
-  type?: string;
-}
-
-export default class Push extends SnapshotBase {
+export default class Push extends Command {
   public static description =
     'Preview, validate and deploy your changes to the destination org';
 
   public static flags = {
-    ...SnapshotBase.flags,
+    target: flags.string({
```

Comment:
I though about it but the description differs between commands
```
The unique identifier of the organization where to send the changes
```
VS
```
The unique identifier of the organization to preview
```

Reply:
It might be better to lose in specificity in the description of the flag names so to show that "it's the same thing"

e.g. if you check `-v` in SFDX-CLI, it's always(-ish) used to describe the dev hub organization:
https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_package.htm

However, in the case of more-than-one org commands, it is true that'd want to have a more specific command, 'cause "The unique identifier of the organization" ain't gonna cut it.